src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -417,23 +419,61 @@ public async Task<JObject> Resolve(InvocationExpressionSyntax method, Dictionary
                     using var commandParamsObjWriter = new MonoBinaryWriter();
                     if (isTryingLinq == 0)
                         commandParamsObjWriter.WriteObj(objectId, context.SdbAgent);
+
                     if (method.ArgumentList != null)
                     {
-                        commandParamsObjWriter.Write((int)method.ArgumentList.Arguments.Count + isTryingLinq);
+                        int passedArgsCnt = method.ArgumentList.Arguments.Count;
+                        int sentToDebuggerParamsCnt = passedArgsCnt;
+                        var methodParamsInfo = new List<ParameterInfo>();
+                        var methodInfo = await context.SdbAgent.GetMethodInfo(methodId, token);
+                        if (methodInfo != null) //null for e.g. ToList()
```

Comment:
Why would it be null for `ToList()`? Is it because it's an extension method?
If so, and we are not supporting that in this PR, then this should fail with a message, if `methodInfo == null`.